libr/core/anal_tp.c

```diff
@@ -412,6 +449,7 @@ static int bb_cmpaddr(const void *_a, const void *_b) {
 }
 
 R_API void r_core_anal_type_match(RCore *core, RAnalFunction *fcn) {
+	eprintf ("r_core_anal_type_match (%s @ 0x%"PFMT64x")\n", fcn->name, fcn->addr);
```

Comment:
r_return_if_fail

Reply:
This is all debug stuff, I am still testing.